index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e746991a9e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```

Comment:
- [ ] Kindly use the `<title>` tag to define the title of the document. The title is shown in the browser's title bar or in the page's tab. The `<title>` tag is required in HTML documents! The contents of a page title are very important for search engine optimization (SEO)! 👩🏽‍🔬